tests/loader/test_argoverse_dataset_loader.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+from pathlib import Path
+
+import numpy as np
+
+import gtsfm.utils.io as io_utils
+from gtsam import Cal3Bundler, Pose3
+from gtsfm.loader.argoverse_dataset_loader import ArgoverseDatasetLoader
+
+ARGOVERSE_DATA_ROOT_PATH = Path(__file__).resolve().parent.parent / "data" / "argoverse"
+
+
+class TestArgoverseDatasetLoader(unittest.TestCase):
+    """Tests a loader that loads Argoverse image and camera data."""
+
+    def setUp(self) -> None:
+        """ """
+        self.loader = ArgoverseDatasetLoader(
+            dataset_dir=ARGOVERSE_DATA_ROOT_PATH / "train1",
+            log_id="273c1883-673a-36bf-b124-88311b1a80be",
+            stride=1,
+            max_num_imgs=2,
+            max_lookahead_sec=50,
+            camera_name="ring_front_center",
```

Comment:
as these params are used in the function `test_get_image_contents`, maybe make them global variables above this class?